### PR TITLE
complete javascript-forms exercise -- Cameron Peck

### PR DESCRIPTION
**Pull request by cam-peck**
https://user-images.githubusercontent.com/105157437/188937365-9bed4e46-ad36-4237-9771-40ec26e4293e.mp4

### CODE REVIEW

File: javascript-forms/form-events.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+$contactForm.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var formData = {};
+  formData.name = $contactForm.elements.name.value;
+  formData.email = $contactForm.elements.email.value;
+  formData.message = $contactForm.elements.message.value;
+  console.log('Message Data: ', formData);
+  $contactForm.reset();
+});
```
Comment (Zhiwutian) [inline javascript-forms/form-events.js:42]: this should all be in the `form-values` js file. The rest of the code should not be commented out

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed!

File: javascript-forms/form-events.js (hunk 1 above)
Comment (cam-peck) [reply javascript-forms/form-events.js:42]: I just re-read the instructions -- oops. On it!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**